rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel = "nightly-2025-04-05" # Make sure to update `template/*/rust-toolchain.toml` as well.
+channel = "nightly-2025-04-04" # Make sure to update `template/*/rust-toolchain.toml` as well.
```

Comment:
why?

Reply:
Some rustc error on mac, will be overridden by #1843